docs/docs/changelog/v8-11.md

```diff
@@ -0,0 +1,19 @@
+---
+slug: v8.11-release
+title: v8.11
+authors: ecalc-team
+tags: [release, eCalc]
+sidebar_position: 20
+---
+
+# eCalc
+
+
+
+## New Features
+
+## Fixes
+
+## Breaking changes
+- Sum and split mechanical power at installation level: The total power is now the sum of mechanical (e.g. compressor with turbine) power and electrical power. Previously, the total power included only electrical power.

```

Comment:
```suggestion
- Sum and split mechanical power at installation level: The total power is now the sum of mechanical (e.g. compressor with turbine) power and electrical power. 
*NB!* This may lead to big changes in power usage, as mechanical power was previously not included in the total!
```

Reply:
? even more emphasized?

Reply:
Done